lib/protocol/networks.js

```diff
@@ -864,9 +864,7 @@ const simnet = {};
 
 simnet.type = 'simnet';
 
-simnet.seeds = [
-  'aorsxa4ylaacshipyjkfbvzfkh3jhh4yowtoqdt64nzemqtiw2whk@127.0.0.1'
-];
+simnet.seeds = [];
 
 simnet.magic = 0x473bd012;
```

Comment:
Should we change the magic number? Prevents any weirdness for anyone who may have a leftover simnet directory in their hsd folder. - OR, thinking out loud here, we write a small script to just nuke any current ~./hsd/simnet folder to prevent any collisions.

Reply:
I don't think we should change the magic number but I'm open to it if you are strongly opinionated

Reply:
We won't have to make any changes for `hnsd` if we don't change the magic